doc/versionHistory.md

```diff
@@ -1,5 +1,12 @@
 # Version History
 
+0.4.5
+
+- Rename **convertToKeyValuePair_integer.vi** for **convertToKeyValuePair_integer32bit.vi**.
+- Rename **convertToKeyValuePair_unsignedInteger.vi** for **convertToKeyValuePair_unsignedInteger32bit.vi**.
+- Create **convertToKeyValuePair_integer64bit.vi**.
+- Create **convertToKeyValuePair_unsignedInteger64bit.vi**.
```

Comment:
- Rename **convertToKeyValuePair_integer.vi** to **convertToKeyValuePair_integer32bit.vi**.
- Rename **convertToKeyValuePair_unsignedInteger.vi** to **convertToKeyValuePair_unsignedInteger32bit.vi**.
- Create **convertToKeyValuePair_integer64bit.vi** and **convertToKeyValuePair_unsignedInteger64bit.vi**.

Reply:
Done. Thanks.